pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoPackage, fetchgit, git }:
 
 buildGoPackage rec {
   version = "2.11.0";
   name = "helm-${version}";
 
-  src = fetchFromGitHub {
-    owner = "helm";
-    repo = "helm";
+  src = fetchgit {
+    url = "https://www.github.com/helm/helm.git";
     rev = "v${version}";
-    sha256 = "1z810a6mxyrrw4i908dip8aqsj95c0kmv6xpb1wwhskg1zmf85wk";
+    sha256 = "1jjyyidffls0yzi00p7icg19f10wdrmcgb52migdy4kv2chvmrah";
+    leaveDotGit = "true";
```

Comment:
The problem about this option is that the checksum can break at any time git decides to change their .git directory layout. Therefore we want to avoid it at all cost.
I would suggest to either write a script instead that generates this information automatically,
like here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/analysis/radare2/default.nix#L92

Or just set the version only, which might be enough in most cases.

Reply:
Better doing that manually then going through the `leaveDotGit` hell.